index.html

```diff
@@ -28,32 +30,55 @@
                     <div class="code highlight">
                         <span class="kr">read</span>
                         <span id="log-example-adapter" class="adapter" style="width: 60px;"></span>
-                        -last <span class="s s-Moment">:1 day:</span> status_code = <span class="mi">500</span>
+                        -last <span class="s s-Moment">:day:</span> status_code=<span class="mi">500</span><br>
                         <span class="o">|</span> <span class="kr">reduce</span> -every
-                        <span class="s s-Moment">:1m:</span> count()
+                        <span class="s s-Moment">:1m:</span> count()<br>
+                        <span class="o">|</span> <span class="kr">view</span> table
                     </div>
                     <p class="description">
                         Find all the log lines with status_code 500 in the last
-                        day and return the per-minute counts - either from
+                        day and return the per-hour counts - either from
                         splunk, a file, or elastic.
                     </p>
                 </div>
                 <div class="juttle-example">
                     <h3>
-                        Moving averages are just one line away:
+                        Mixed historical / live moving averages are just one line away:
                     </h3>
                     <div class="code highlight">
                         <span class="kr">read</span>
                         <span id="moving-average-example-adapter" class="adapter" style="width: 80px;"></span>
-                        -last <span class="s s-Momemnt">:6 hours:</span>
+                        -from <span class="s s-Momemnt">:1 hour ago:</span> -to <span class="s s-Momemnt">:end:</span>
```

Comment:
Fixed both suggestions.


On Wed, Mar 16, 2016 at 11:41 AM, Daria Mehra <notifications@github.com>
wrote:

> In index.html
> <https://github.com/juttle/juttle.github.io/pull/5#discussion_r56391690>:
>
> >                      </h3>
> >                      <div class="code highlight">
> >                          <span class="kr">read</span>
> >                          <span id="moving-average-example-adapter" class="adapter" style="width: 80px;"></span>
> > -                        -last <span class="s s-Momemnt">:6 hours:</span>
> > +                        -from <span class="s s-Momemnt">:1 hour ago:</span> -to <span class="s s-Momemnt">:end:</span>
>
> it's not really called Momemnt is it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/juttle/juttle.github.io/pull/5/files/77e6eb12ba5028d26ff9e662aa310101a43a4cde#r56391690>
>



-- 
Michael Demmer
Jut - CTO / VP Engineering
demmer@jut.io